units/src/locktime/absolute.rs

```diff
@@ -607,4 +607,34 @@ mod tests {
         timestamps.reverse();
         assert_eq!(MedianTimePast::new(timestamps).unwrap().to_u32(), 500_000_005);
     }
+
+    #[test]
+    fn height_is_satisfied_by() {
+        let chain_tip = Height::from_u32(100).unwrap();
+        let locktime = Height::from_u32(100).unwrap();
+        assert!(locktime.is_satisfied_by(chain_tip));
+
+        let chain_tip = Height::from_u32(100).unwrap();
+        let locktime = Height::from_u32(101).unwrap();
+        assert!(locktime.is_satisfied_by(chain_tip));
+
+        let chain_tip = Height::from_u32(100).unwrap();
+        let locktime = Height::from_u32(102).unwrap();
+        assert!(!locktime.is_satisfied_by(chain_tip));
```

Comment:
`chain_tip` is the same in each assertion so I rekon only define it once. And can we have a comment on the middle test case

```
// lock is satisfied if transaction can go in the next block.
```